File: chapter_one/UN_CC_ASSIGNMENTS/INPROG01/SECLAR.py
# https://www.codechef.com/INPRG01/problems/SECLAR
A = int(input())
B = int(input())
C = int(input())

if (B >= A) and (A >= C) and (B >= C):
    ln = A
if (C >= A) and (A >= B) and (C >= B):
    ln = A

if (A >= B) and (B >= C) and (A >= C):
    ln = B
if (C >= B) and (B >= A) and (C >= A):
    ln = B

if (A >= C) and (C >= B) and (A >= B):
    ln = C
if (B >= C) and (C >= A) and (B >= A):
    ln = C


print(ln)

# (c) 2021, Vikramaditya V. Vajire
# https://github.com/vikramaditya
# This source code is licensed under the MIT License

File: chapter_one/UN_CC_ASSIGNMENTS/INPROG01/SECLAR.c
// https://www.codechef.com/INPRG01/problems/SECLAR

#include <stdio.h>
int main(){
  int A;
  int B;
  int C;
  int L;

  scanf("%d %d %d", &A, &B, &C);

  if (B >= A && A >= C && B >= C){
    L = A;
  }

  if (C >= A && A >= B && C >= B){
    L = A;
  }

  if (A >= B && B >= C && A >= C){
    L = B;
  }

  if (C >= B && B >= A && C >= A){
    L = B;
  }

  if (A >= C && C >= B && A >= B){
    L = C;
  }

  if (B >= C && C >= A && B >= A){
    L = C;
  }

  printf("%d", L);

  return 0 ;

}
